README.md

```diff
@@ -36,36 +38,35 @@ export PYTHONPATH=`pwd`:$PYTHONPATH
 alias atomicapp="python `pwd`/atomicapp/cli/main.py"
 ```
 
-### Build
+### Build an Atomic App
```

Comment:
Can we be more clear about what this is doing? "Build an Atomic App" sounds like I am doing an application build (i.e. helloapache) vs packaging up the atomicapp software. Maybe this would be better: `Build an Atomic App Container Image - for Container Execution`. And then mention running it via Atomic CLI? 

And maybe we even want to change the name of the previous section to `Install Atomic App - for Local Execution`